content/docs/tasks/telemetry/querying-metrics/index.md

```diff
@@ -91,14 +91,16 @@ Prometheus server that comes preconfigured to scrape Mixer endpoints to collect
 the exposed metrics. It provides a mechanism for persistent storage and querying
 of Istio metrics.
 
-The configured Prometheus add-on scrapes three endpoints:
+The configured Prometheus add-on scrapes the following endpoints:
 
-1. *istio-mesh* (`istio-mixer.istio-system:42422`): all Mixer-generated mesh
+1. *istio-mesh* (`istio-telemetry.istio-system:42422`): all Mixer-generated mesh
```

Comment:
Why are these not capitalized and separated. Is that the actual value of the endpoint? That case seems unlikely since I see the actual code in the parenthesis. Please capitalize appropriately and write them as separate words. Please use verbs and prepositions to make the actions explicit:
```suggestion
1. **Istio mesh** addressable at `istio-telemetry.istio-system:42422`: Returns all Mixer-generated mesh
```

Reply:
Then, please refer to them as jobs and link to that page in your content. For example: The `istio-mesh` job ...

Reply:
The page is used to tell the user how to verify the prometheus configurations. I do not think it is proper to link that here. How about change to - `istio-telemetry.istio-system:42422`: The `istio-mesh` job returns all Mixer-generated mesh